### PR TITLE
added web accesibility question

### PR DESCRIPTION
**Pull request by lishakothari**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -186,8 +187,22 @@ const accessibilityQuiz = [
...
+    Answer:
+      "ARIA defines the responsibilities, characteristics, and states for screenreaders in order to convey information about custom modules like carousels, accordions, and tabs.",
+    Distractor1:
+      "A tool that changes visuals on a web page to make it more accesible.",
+    Distractor2:
+      "It is an open-source, automated web page quality measurement tool.",
+    Distractor3:
+      "A computer program to scan an application for accessibility improvements.",
+    Explanation:
+      "ARIA (Accessible Rich Internet Application) is the acronym for Accessible Rich Internet Application. Screenreader users will benefit from ARIA.ARIA defines the responsibilities, characteristics, and states for screenreaders in order to convey information about such modules. In a nutshell, ARIA serves as a link between HTML's missing information and screenreaders.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:203]: This is a good explanation but let's shorten it.
Explanations work best if they are 1-2 sentences long. If the user wants to learn more, then they can click on the learn more link. 

```suggestion
      "ARIA (Accessible Rich Internet Application) is the acronym for Accessible Rich Internet Application. ARIA defines the responsibilities, characteristics, and states for screen readers in order to convey information about such modules.",
```

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -186,8 +187,22 @@ const accessibilityQuiz = [
       "A computer program to scan an application for accessibility improvements.",
     Explanation:
       "A software application with assistive technology that attempts to convey what people with normal eyesight see on a display to their users via non-visual means, like text-to-speech. Developers can utilize a screen reader to understand how a webpage can be made accessible to such people and develop the page accordingly.",
-    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
   },
+  {
+    Question: "What is ARIA?",
+    Answer:
+      "ARIA defines the responsibilities, characteristics, and states for screenreaders in order to convey information about custom modules like carousels, accordions, and tabs.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:195]: Good but let's shorten it. shorter answers tend to work better in the quiz
```suggestion
      "ARIA defines characteristics for screen readers in order to convey information about custom modules like carousels, and tabs.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/accessibility-quiz.ts (hunk 1 above)
Comment (lishakothari) [reply src/data/accessibility-quiz.ts]: Please Check! :)

File: src/data/accessibility-quiz.ts (hunk 2 above)
Comment (lishakothari) [reply src/data/accessibility-quiz.ts]: Kindly check if its proper now :)

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**